1209/42B0/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: RP2040 0.42 LCD
+owner: 01space
```

Comment:
Please have yourself or "CircuitPython" as the owner so that it is clear that this PID wasn't granted to the manufacturer.

Reply:
Ah, so should I not create the org then either ?

Reply:
Make one for yourself or CP but not the manufacturer.